test/distributed/test_distributed.py

```diff
@@ -2113,7 +2113,7 @@ def test_DistributedDataParallel_non_default_stream(self):
                 # Clear gradients manually
                 grad = net.module.weight.grad
                 if grad is not None:
-                    grad.detach_()
+                    grad.requires_grad_(False)
```

Comment:
This is not equivalent as it won't clear the grad_fn if one exists.
It may be ok for this test though.
Is there a reason why you don't use the `zero_grad()` function from the Module?

Reply:
I guess the author just wanted to set one grad in this module, not all grads in the module, so trying to follow the original style in this test